src/pages/SpacesDashboard/index.ts

```diff
@@ -16,7 +15,5 @@ export const SpacesDashboard = compose(
   withRequired({
     required: ["worldId", "userId", "world"],
     fallback: LoadingPage,
-  }),
-  withOwnSpaces,
-  withRequired({ required: ["ownSpaces"], fallback: LoadingPage })
```

Comment:
Why not use loading state while data is being loaded?

Reply:
This data is no longer used in SpacesDashboard